iib/workers/tasks/build_create_empty_index.py

```diff
@@ -0,0 +1,152 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+import logging
+import stat
+import tempfile
+import json
+import re
+
+from iib.workers.api_utils import set_request_state
+from iib.workers.tasks.build import (
+    _add_label_to_index,
+    _build_image,
+    _cleanup,
+    _create_and_push_manifest_list,
+    _get_index_database,
+    _opm_index_rm,
+    _push_image,
+    _serve_index_registry,
+    _update_index_image_build_state,
+    _update_index_image_pull_spec,
+)
+from iib.workers.tasks.celery import app
+from iib.workers.tasks.utils import (
+    chmod_recursively,
+    request_logger,
+    prepare_request_for_build,
+    run_cmd,
+    RequestConfigCreateIndexImage,
+)
+
+__all__ = ['handle_create_empty_index_request']
+
+log = logging.getLogger(__name__)
+
+
+def _get_present_operators(from_index, base_dir):
+    """
+    Get a list of operators already present in the index image.
+
+    :param str from_index: index image to inspect.
+    :param str base_dir: base directory to create temporary files in.
+    :return: list of unique present operators as provided by the grpc query
+    :rtype: list, list
```

Comment:
The function actually only returns a single list. Is this a typo?

Reply:
good catch